index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+	<head>
+	
+		<title>Don Quixote</title>
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+	
+	</head>
+	
+	<body>
+	
+		<img src="images/DON_QUIXOTE.jpg">
+		
+		<nav>
+			<a href="index.html" STYLE="text-decoration: none">About me</a> |
+			<a href="interests.html">My experience</a> |
+			<a href="http://www.mcnallyjackson.com/book/9781632060754" target="_blank">Buy the book</a>
+		</nav>
+		
+		<h2>the great</h2> 
+		
+		<h1>Don Quixote</h1>
+
+		<h4>
+		I do not see the world for what it is and prefer <br>to imagine that I am living out a knightly story.
+		</h4>
+		
+		<p>
+		I'm a nameless hidalgo (member of the lesser Spanish nobility), almost 50 years old, who reads so many chivalric romances that I lose my sanity and decide to set out to revive chivalry, undo wrongs and bring justice to the world under the name <b>Don Quixote</b>. 
+		</p>
+
+		<p>
+		I recruit a simple farmer, <b>Sancho Panza</b>, as my squire, who often employs a unique, earthy wit in dealing with my rhetorical orations on antiquated knighthood. 
```

Comment:
:hammer: The `<b>` tag is no longer used, although it works. Wrap the text in a span with a class and style the class instead:
```html
...<span class="sancho">Sancho Panza</span>...
```